components/picker/index.css

```diff
@@ -230,39 +219,30 @@ governing permissions and limitations under the License.
     pointer-events: none;
     content: '';
     position: absolute;
+    block-size: calc(100% + (var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) * 2)
+                    + (var(--mod-picker-border-width, var(--spectrum-picker-border-width)) * 2));
+
+    inline-size: calc(100% + (var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) * 2)
+                    + (var(--mod-picker-border-width, var(--spectrum-picker-border-width)) * 2));
+
+    margin-block-start: calc((var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap))
+                            + var(--mod-picker-focus-indicator-thickness, var(--spectrum-picker-focus-indicator-thickness))
+                            + var(--mod-picker-border-width, var(--spectrum-picker-border-width))) * -1);
+
+    margin-inline-start: calc((var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap))
+                            + var(--mod-picker-focus-indicator-thickness, var(--spectrum-picker-focus-indicator-thickness))
+                            + var(--mod-picker-border-width, var(--spectrum-picker-border-width))) * -1);
     left: 0;
     right: 0;
     bottom: 0;
     top: 0;
```

Comment:
As a question for other days, if you used `inset-inline` could you get away with one rule instead of _both of_ `margin-inline-start` AND `inline-size`?

Reply:
It looks like I can do:
```css
inset-inline: 0;
inset-block: 0;
```

instead of the four, and it'll work the same.